examples/k8s-jobs-log-selector.yaml

```diff
@@ -0,0 +1,47 @@
+# This example demonstrates how to include and obtain logs from pods created by a
+# custom resource submitted via resource template.
+#
+# This is particularly useful since Argo Workflows does not know how
+# other CRDs (Kubeflow training CRDs, Spark application CRD, etc.) work
+# and thus could not pull the logs from the pods created by those CRDs.
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: k8s-jobs-log-selector-
+spec:
+  entrypoint: tf-jobtmpl
+  templates:
+  - name: tf-jobtmpl
+    resource:
+      action: create
+      successCondition: status.succeeded = 2
+      failureCondition: status.failed > 0
+      # You can also create any other custom K8s resource here. This is an example
+      # of using Kubeflow TFJob as an illustration.
+      manifest: |
+        apiVersion: kubeflow.org/v1
+        kind: TFJob
+        metadata:
+          name: tfjob-examples
+        spec:
+          tfReplicaSpecs:
+             Worker:
+               replicas: 2
+               restartPolicy: Never
+               template:
+                 metadata:
+                   # We add this label to the pods created by TFJob custom resource to inform Argo Workflows
+                   # that we want to include the logs from the created pods. Once the pods are created with this
+                   # label, you can then use `argo logs -c tensorflow -l <logs-label-selector>` to the logs from
```

Comment:
Is the label selector necessary, or do you get the logs for free just because they're labeled as this workflow?

Reply:
Good catch. This is optional. I moved this to the second paragraph.